src/Illuminate/Foundation/helpers.php

```diff
@@ -657,9 +657,10 @@ function event($event, $payload = array(), $halt = false)
 	* Get the path to a versioned Elixir file.
 	*
 	* @param  string  $file
+	* @param  string  $default
```

Comment:
Well I don't know :P I was trying to intepret Graham's message, sorry for not being very helpful after all. 

But seriously, it should be `mixed` no?